.github/workflows/build.yaml

```
@@ -13,26 +13,27 @@ jobs:
-        
-    - name: Debug lockfile
-      run: ls -la ${{ github.workspace }}
+        version: 10.17.1
 
+    # 3. Cache the pnpm store
     - name: Get pnpm store directory
-      shell: bash
-      run: |
-        echo "STORE_PATH=$(pnpm store path --silent)" >> $GITHUB_ENV
-
+        shell: bash
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 33: mapping values are not allowed in this context

(syntax-check)